autogen.sh

```diff
@@ -75,8 +75,6 @@ gen_executables() {
     done
 }
 
-./script/gen-ffi-macros
-
```

Comment:
In this PR I removed the script previously used to generate FFI macros. These macros now are not used anymore and we're using instead a more understandable approach based on using `jinja2`.